lib/modules/configuration/bloc/mission_controller/variables_bloc.dart

```diff
@@ -0,0 +1,44 @@
+import 'package:bloc/bloc.dart';
+import 'package:zenith_monitor/utils/mixins/class_mission_variables.dart';
+import 'package:zenith_monitor/pipelines/mission_pipeline/mission_bloc.dart';
+
+part 'variables_state.dart';
+part 'variables_event.dart';
+
+/// This bloc has to do with the creation of current misson variables.
+/// So, it deals with user interation in mission_creation.dart widget
+/// and update the MissionVariableList object in mission pipeline.
+
+class VariablesBloc extends Bloc<VariablesEvent, VariablesState> {
```

Comment:
Vou fazer uns comentários meio opcionais de nomenclatura, não necessáriamente devem ser feitas modificações e pode ignorar se achar "exagero":

Será que aqui não ficaria mais descritivo o nome se for `MissionVariablesBloc`? Apenas `VariablesBloc` talvez fique muito genérico e pode parecer que o bloc tem um escopo maior do que realmente tem, também segue o mesmo "padrão" de outras nomenclaturas usadas como `MissionBloc` e `MissionVariablesList`

Reply:
Concordo também,  o que você acha de ```MissionVariablesController```? Já que temos essa ideia de que **controller** são os blocs que lidam com a UI. Poderíamos manter o padrão de especificar o nome dos blocs entre **pipeline** e **controller**.